### PR TITLE
Update assign-role-permissions.md to clarify assignment instructions.

### PR DESCRIPTION
**Pull request by leswaters**
Users are confused by the absence of **Manage team** when selecting it from a pinned team. This stems from the fact that teams themselves are not pinne. Rather, only channels are pinned. The revised documentation clarifies this by instructing the user to choose the team name under *Your teams* rather than under *pinned*. In addition, the ellipses (**...**) are parenthetically added to **More options** for further clarification.

### CODE REVIEW

File: Teams/assign-roles-permissions.md [hunk 1]
```diff
@@ -62,7 +62,10 @@ For more information about moderator capabilities, see [Set up and manage channe
 
 ## Assign a user role
 
-To assign a user role, in Teams, select the team name and click **More options** > **Manage team**. On the **Members** tab, you can add members and choose owners and moderators (if you have sufficient permissions). For more information , see [Change team settings in Teams](https://support.office.com/article/ce053b04-1b8e-4796-baa8-90dc427b3acc).
+To assign a user role, in Teams, select the team name and click **More options** (**...**) > **Manage team**. On the **Members** tab, you can add members and choose owners and moderators (if you have sufficient permissions). For more information , see [Change team settings in Teams](https://support.office.com/article/ce053b04-1b8e-4796-baa8-90dc427b3acc).
```
Comment (JohanFreelancer9) [inline Teams/assign-roles-permissions.md:65]:
```suggestion
To assign a user role, in Teams, select the team name and then select **More options** (**...**) > **Manage team**. On the **Members** tab, you can add members and choose owners and moderators (if you have sufficient permissions). For more information, see [Manage team settings and permissions in Teams](https://support.office.com/article/ce053b04-1b8e-4796-baa8-90dc427b3acc).
```

File: Teams/assign-roles-permissions.md [hunk 2]
```diff
@@ -62,7 +62,10 @@ For more information about moderator capabilities, see [Set up and manage channe
...
+
+> [!NOTE]
+> The **Manage team** option will not appear for pinned channels. Select the team name under *Your teams* further below and click **More options** (**...**) to the right of the name.
```
Comment (JohanFreelancer9) [inline Teams/assign-roles-permissions.md:68]:
```suggestion
> The **Manage team** option will not appear for pinned channels. Select the team name under *Your teams* further below and then select **More options** (**...**) to the right of the name.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @leswaters, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**